src/platform/packages/shared/kbn-unified-field-list/src/components/field_list_grouped/fields_accordion.tsx

```diff
@@ -69,7 +69,15 @@ function InnerFieldsAccordion<T extends FieldListItem = DataViewField>({
 
     return (
       <EuiText size="xs">
-        <strong className={titleClassname}>{label}</strong>
+        <strong
+          className={titleClassname}
+          aria-label={i18n.translate('unifiedFieldList.fieldsAccordion.accordionButtonAriaLabel', {
+            defaultMessage: '{label}: {fieldsCount} items',
```

Comment:
```suggestion
            defaultMessage: '{label}: {fieldsCount} {fieldsCount, plural, one {item} other {items}}',
```
I think this is what @alexwizp was referring to re pluralization. We can use this syntax to toggle item(s) based on field count.

Reply:
Improved in [9428dfd](https://github.com/elastic/kibana/pull/216993/commits/9428dfd5de2903eb8bd5a4331a14bb6ae3611e18)

Reply:
No worries, the syntax isn't obvious when you first encounter it (at least it wasn't for me 😅).